Sources/SwiftPlot/PlotStyleHelpers.swift

```diff
@@ -52,18 +52,34 @@ struct Text : Annotation {
     public var color = Color.black
     public var size: Float = 15
     public var location = Point(0.0, 0.0)
+    public var bbox: Bool = false
```

Comment:
@KarthikRIyer would changing ```bbox``` to ```drawBoundingBox``` be applied to the other variables as well? for example, would ```bboxBorderSize``` and ```bboxColor``` be changed to ```boundingBoxBorderSize``` and ```boundingBoxColor``` instead?

Reply:
Currently I'm leaving them unchanged, but if you think otherwise let me know. Thanks!

Reply:
Yes, those should be changed to the more descriptive names.